great_expectations/cli/datasource.py

```diff
@@ -447,15 +452,181 @@ def _add_spark_datasource(context, prompt_for_datasource_name=True):
     context.add_datasource(name=data_source_name, class_name='SparkDFDatasource', **configuration)
     return data_source_name
 
+def select_datasource(context, data_source_name=None):
```

Comment:
param `data_source_name` -> `datasource_name`

Reply:
Changed to datasource_name everywhere in the cli